pkg/restapi/restapi.go

```diff
@@ -30,6 +30,9 @@ const (
 	// ParameterAccountID is used to identify an account by its ID.
 	ParameterAccountID = "accountID"
```

Comment:
this can be removed. Please also adapt `RouteValidatorsAccount` to use an address